test/controllers/releases_controller_test.rb

```diff
@@ -6,17 +6,23 @@
 describe ReleasesController do
   let(:project) { projects(:test) }
   let(:release) { releases(:test) }
+  let(:release_dot) { releases(:test_dot) }
```

Comment:
prefer modifying an existing release instead of adding a new fixture every test has to load